src/plugins/kibana/public/settings/sections/indices/add_data_steps/pipeline_setup/directives/pipeline_setup.js

```diff
@@ -0,0 +1,87 @@
+import uiModules from 'ui/modules';
+import _ from 'lodash';
+import Pipeline from '../lib/pipeline';
+import Processor from '../lib/processor';
+import angular from 'angular';
+import processorTypes from '../lib/processor_types';
+import savedPipeline from '../sample_pipeline.json'; //temp for debugging purposes
```

Comment:
Can the sample pipeline go away now? Especially since this PR only includes the set processor, I don't think the sample works